cmd/nodelink-controller/main.go

```diff
@@ -61,15 +60,15 @@ const (
 	controllerName = "nodelink"
 
 	// machineAnnotationKey is the annotation storing a link between a node and it's machine. Should match upstream cluster-api machine controller. (node.go)
-	machineAnnotationKey = "cluster.k8s.io/machine"
+	machineAnnotationKey = "machine.k8s.io/machine"
```

Comment:
This seems a little weird. I feel like we should either keep this the same as upstream, or go ahead and change it to `openshift.io`.

Reply:
sure, it was meant to be `machine.openshift.io`, thanks!